### ISSUE TITLE
MNK-type Mobs not double attacking once Charmed

### ISSUE THREAD

**Issue by BaldrDash** [DarkstarProject/darkstar#922]
Mandragora, and Sabotender will not use their signature double attacks after being Charmed by a BST, though I imagine this applies to all mobs listed as MNK types.

**Comment by Scavenge** [comment]
just noticed this myself, also not sure but it seems like the mandragoras will counter, but only when charmed; i'm not sure i've seen an uncharmed mandragora use counter.

**Comment by maxtherabbit** [comment]
That's correct, (uncharmed) mobs don't counter atm. It's on my list to implement
